alarm/Exception.js

```diff
@@ -177,12 +187,12 @@ module.exports = class {
           }
         }
         let valArray = val, val2Array = val2;
-        this.isJsonString(val) && (valArray = JSON.parse(val));
-        this.isJsonString(val2) && (val2Array = JSON.parse(val2));
+        isJsonString(val) && (valArray = JSON.parse(val));
+        isJsonString(val2) && (val2Array = JSON.parse(val2));
         if (key && key.startsWith("p.tag.ids")) {
           valArray = valArray.map(Number);
```

Comment:
this is no longer needed as it's already number at exception creation
We should make sure it's number in alarm as well.

or, actually, what's the point to convert it to number. maybe it's not necessary at all?